extension/src/cli/dvc/contract.ts

```diff
@@ -233,10 +233,16 @@ export type TemplatePlotOutput = {
   type: PlotsType
 }
 
+export type BoundingBox = {
+  box: { left: number; right: number; top: number; bottom: number }
+  score: number
+}
+
 export type ImagePlotOutput = {
   revisions: string[]
   type: PlotsType
   url: string
+  annotations?: { [label: string]: BoundingBox[] }
```

Comment:
DVC spec has been changed from `boxes` to `annotations`. I've updated the code accordingly.